schemas/device/MRIScanner.schema.tpl.json

```diff
@@ -0,0 +1,20 @@
+{
+  "_type": "https://openminds.ebrains.eu/MRI/MRIScanner",
+  "_extends": "/core/schemas/products/device.schema.tpl.json",
+  "properties": {
+    "magneticFieldStrength": {
+      "_instruction": "Enter the nominal field strength of MR magnet in Tesla.",
+      "_embeddedTypes": [
+        "https://openminds.ebrains.eu/core/QuantitativeValue"
+      ]
+    },
+    "stationName": {
+      "type": "string",
+      "_instruction": "Enter institution defined name of this Scanner"
+    },
+    "receiveCoilName": {
```

Comment:
Agreed on both moving it to MRIScannerUsage and renaming it to `radiofrequencyCoilType` or `RFCoil` while making it a controlled term.

On the issue of the `transmitter coil`: I agree that if we are going to have receiver coil, it is logical to have a transmitter too. However, I rarely see people report that one, as coils are either receiver-emitter (transceiver) or receiver coil, while the body coil of the scanner is used as the transmitter. Anyway, renaming it would solve the problem as they can specify the transmitter coil if it is present.